### PR TITLE
Fix quick start link.

### PR DESCRIPTION
**Pull request by di-wu**
**Overview**
Fixed broken Quick Start link in README.

### CODE REVIEW

**Comment by di-wu** [comment]
This link is broken on multiple locations.

In `docs/modules/developers-guide/pages/tutorials-intro.adoc` for example:

```
../quickstart/quickstart{outfilesuffix}[Quick start]
```

Do I include fixes for these as well?

File: modules/ROOT/pages/videos-tutorials.adoc [hunk 1]
```diff
@@ -65,7 +65,7 @@ video::yqIoiyuGYNA[youtube,width=100%,height=480px]
 
 This video segment walks through how to deploy your first application with the {IC} running locally.
 
-To try it yourself, follow the steps in the link:quickstart/quickstart{outfilesuffix}[Quick start] tutorial.
+To try it yourself, follow the steps in the link:quickstart/quickstart-intro{outfilesuffix}[Quick start] tutorial.
```
Comment (lsgunnlsgunn) [inline modules/ROOT/pages/videos-tutorials.adoc:68]:
```suggestion
To try it yourself, follow the steps in the link:quickstart/local-quickstart{outfilesuffix}[Quick start] tutorial.
```

File: modules/ROOT/pages/videos-tutorials.adoc [hunk 2]
```diff
@@ -167,7 +167,7 @@ To be part of the conversation, join the  link:forum.dfinity.org[Developer Forum
 
 == Tutorials
 
-The link:quickstart/quickstart{outfilesuffix}[Quick start] provides a simplified introduction to the basic work flow for creating and deploying a new project without exploring the contents of the project directory or sample code.
+The link:quickstart/quickstart-intro{outfilesuffix}[Quick start] provides a simplified introduction to the basic work flow for creating and deploying a new project without exploring the contents of the project directory or sample code.
```
Comment (lsgunnlsgunn) [inline modules/ROOT/pages/videos-tutorials.adoc:170]:
```suggestion
The link:quickstart/local-quickstart{outfilesuffix}[Quick start] provides a simplified introduction to the basic work flow for creating and deploying a new project without exploring the contents of the project directory or sample code.
```

File: modules/developers-guide/pages/install-upgrade-remove.adoc [hunk 3]
```diff
@@ -7,7 +7,7 @@
 :sdk-short-name: DFINITY Canister SDK
 :sdk-long-name: DFINITY Canister Software Development Kit (SDK)
 
-As described in the link:../quickstart/quickstart{outfilesuffix}[Quick start], you can download and install the latest version of the {sdk-short-name} package by running a command in a terminal shell.
+As described in the link:../quickstart/quickstart-intro{outfilesuffix}[Quick start], you can download and install the latest version of the {sdk-short-name} package by running a command in a terminal shell.
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/install-upgrade-remove.adoc:10]:
```suggestion
As described in the link:../quickstart/local-quickstart{outfilesuffix}[Quick start], you can download and install the latest version of the {sdk-short-name} package by running a command in a terminal shell.
```

File: modules/developers-guide/pages/lang-service-ide.adoc [hunk 4]
```diff
@@ -28,7 +28,7 @@ You can use any editor or integrated development environment that supports the L
 For example, if you use Visual Studio Code (VSCode) or Emacs as your development environment, you can install a plugin extension that enables those editors to automatically invoke the {proglang} language server.
 
 NOTE: Only the Visual Studio Code (VSCode) plugin extension is currently available for {proglang}.
-For information about installing the plugin, see link:../quickstart/quickstart{outfilesuffix}#install-vscode[Install the language editor plug-in].
+For information about installing the plugin, see link:../quickstart/quickstart-intro{outfilesuffix}#install-vscode[Install the language editor plug-in].
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/lang-service-ide.adoc:31]:
```suggestion
For information about installing the plugin, see link:../quickstart/local-quickstart{outfilesuffix}#install-vscode[Install the language editor plug-in].
```

File: modules/developers-guide/pages/tutorials/access-control.adoc [hunk 5]
```diff
@@ -32,9 +32,9 @@ image::principal-identities.svg[]
 
 Before starting the tutorial, verify the following:
 
-* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart{outfilesuffix}#download-and-install[Download and install].
+* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart-intro{outfilesuffix}#download-and-install[Download and install].
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/access-control.adoc:35]:
```suggestion
* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/local-quickstart{outfilesuffix}#download-and-install[Download and install].
```

File: modules/developers-guide/pages/tutorials/access-control.adoc [hunk 6]
```diff
@@ -32,9 +32,9 @@ image::principal-identities.svg[]
...
 
-* You have installed the Visual Studio Code plugin for {proglang} as described in link:../../quickstart/quickstart{outfilesuffix}#install-vscode[Install the language editor plug-in] if you are using Visual Studio Code as your IDE.
+* You have installed the Visual Studio Code plugin for {proglang} as described in link:../../quickstart/quickstart-intro{outfilesuffix}#install-vscode[Install the language editor plug-in] if you are using Visual Studio Code as your IDE.
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/access-control.adoc:37]:
```suggestion
* You have installed the Visual Studio Code plugin for {proglang} as described in link:../../quickstart/local-quickstart{outfilesuffix}#install-vscode[Install the language editor plug-in] if you are using Visual Studio Code as your IDE.
```

File: modules/developers-guide/pages/tutorials/calculator.adoc [hunk 7]
```diff
@@ -30,7 +30,7 @@ The `+div+` function also includes code to prevent the program from attempting t
 
 Before starting the tutorial, verify the following:
 
-* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart{outfilesuffix}#download-and-install[Download and install].
+* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart-intro{outfilesuffix}#download-and-install[Download and install].
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/calculator.adoc:33]:
```suggestion
* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/loal-quickstart{outfilesuffix}#download-and-install[Download and install].
```

File: modules/developers-guide/pages/tutorials/counter-tutorial.adoc [hunk 8]
```diff
@@ -42,7 +42,7 @@ As the diagram suggests, the network infrastructure and the development environm
 
 Before starting the tutorial, verify the following:
 
-* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart{outfilesuffix}#download-and-install[Download and install].
+* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart-intro{outfilesuffix}#download-and-install[Download and install].
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/counter-tutorial.adoc:45]:
```suggestion
* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/local-quickstart{outfilesuffix}#download-and-install[Download and install].
```

File: modules/developers-guide/pages/tutorials/custom-frontend.adoc [hunk 9]
```diff
@@ -27,9 +27,9 @@ Before starting the tutorial, verify the following:
 * You have `+node.js+` installed for front-end development and can install packages using `+npm install+` in your project.
 For information about installing node for your local operating system and package manager, see the link:https://nodejs.org/en/[Node] website.
 
-* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart{outfilesuffix}#download-and-install[Download and install].
+* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart-intro{outfilesuffix}#download-and-install[Download and install].
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/custom-frontend.adoc:30]:
```suggestion
* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/local-quickstart{outfilesuffix}#download-and-install[Download and install].
```

File: modules/developers-guide/pages/tutorials/custom-frontend.adoc [hunk 10]
```diff
@@ -27,9 +27,9 @@ Before starting the tutorial, verify the following:
...
 
-* You have installed the Visual Studio Code plugin for {proglang} as described in link:../../quickstart/quickstart{outfilesuffix}#install-vscode[Install the language editor plug-in] if you are using Visual Studio Code as your IDE.
+* You have installed the Visual Studio Code plugin for {proglang} as described in link:../../quickstart/quickstart-intro{outfilesuffix}#install-vscode[Install the language editor plug-in] if you are using Visual Studio Code as your IDE.
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/custom-frontend.adoc:32]:
```suggestion
* You have installed the Visual Studio Code plugin for {proglang} as described in link:../../quickstart/local-quickstart{outfilesuffix}#install-vscode[Install the language editor plug-in] if you are using Visual Studio Code as your IDE.
```

File: modules/developers-guide/pages/tutorials/define-an-actor.adoc [hunk 11]
```diff
@@ -11,14 +11,14 @@ ifdef::env-github,env-browser[:outfilesuffix:.adoc]
...
-In the link:../../quickstart/quickstart{outfilesuffix}[Quick start], you had your first look at a simple program for the {IC} involving an actor object and asynchronous messaging.
+In the link:../../quickstart/quickstart-intro{outfilesuffix}[Quick start], you had your first look at a simple program for the {IC} involving an actor object and asynchronous messaging.
 As the next step in learning to write programs that take advantage of actor-based messaging, this tutorial illustrates how to modify a traditional `+Hello, World!+` program to define an actor, then deploy and test your program on a local network.
 
 == Before you begin
 
 Before starting the tutorial, verify the following:
 
-* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart{outfilesuffix}#download-and-install[Download and install].
+* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart-intro{outfilesuffix}#download-and-install[Download and install].
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/define-an-actor.adoc:21]:
```suggestion
* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/local-quickstart{outfilesuffix}#download-and-install[Download and install].
```

File: modules/developers-guide/pages/tutorials/explore-templates.adoc [hunk 12]
```diff
@@ -25,17 +25,17 @@ Before you start this tutorial, verify the following:
 
 * You have `+node.js+` installed if you want to include the default template files for front-end development in your project.
 
-* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart{outfilesuffix}#download-and-install[Download and install].
+* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart-intro{outfilesuffix}#download-and-install[Download and install].
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/explore-templates.adoc:28]:
```suggestion
* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/local-quickstart{outfilesuffix}#download-and-install[Download and install].
```

File: modules/developers-guide/pages/tutorials/explore-templates.adoc [hunk 13]
```diff
@@ -25,17 +25,17 @@ Before you start this tutorial, verify the following:
...
 
-* You have installed the Visual Studio Code plugin for {proglang} as described in link:../../quickstart/quickstart{outfilesuffix}#install-vscode[Install the language editor plug-in] if you are using Visual Studio Code as your IDE.
+* You have installed the Visual Studio Code plugin for {proglang} as described in link:../../quickstart/quickstart-intro{outfilesuffix}#install-vscode[Install the language editor plug-in] if you are using Visual Studio Code as your IDE.
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/explore-templates.adoc:30]:
```suggestion
* You have installed the Visual Studio Code plugin for {proglang} as described in link:../../quickstart/local-quickstart{outfilesuffix}#install-vscode[Install the language editor plug-in] if you are using Visual Studio Code as your IDE.
```

File: modules/developers-guide/pages/tutorials/hello-location.adoc [hunk 14]
```diff
@@ -19,7 +19,7 @@ This tutorial illustrates how to pass arguments on the command-line in a termina
 
 Before you start this tutorial, verify the following:
 
-* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart{outfilesuffix}#download-and-install[Download and install].
+* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart-intro{outfilesuffix}#download-and-install[Download and install].
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/hello-location.adoc:22]:
```suggestion
* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/local-quickstart{outfilesuffix}#download-and-install[Download and install].
```

File: modules/developers-guide/pages/tutorials/intercanister-calls.adoc [hunk 15]
```diff
@@ -33,7 +33,7 @@ The LinkedUp sample application is implemented using the following canisters:
 
 Before building the sample application, verify the following:
 
-* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart{outfilesuffix}#download-and-install[Download and install].
+* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart-intro{outfilesuffix}#download-and-install[Download and install].
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/intercanister-calls.adoc:36]:
```suggestion
* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/local-quickstart{outfilesuffix}#download-and-install[Download and install].
```

File: modules/developers-guide/pages/tutorials/multiple-actors.adoc [hunk 16]
```diff
@@ -35,7 +35,7 @@ This code sample simply assigns a variable and prints messages for demonstration
 
 Before starting the tutorial, verify the following:
 
-* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart{outfilesuffix}#download-and-install[Download and install].
+* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart-intro{outfilesuffix}#download-and-install[Download and install].
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/multiple-actors.adoc:38]:
```suggestion
* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/local-quickstart{outfilesuffix}#download-and-install[Download and install].
```

File: modules/developers-guide/pages/tutorials/my-contacts.adoc [hunk 17]
```diff
@@ -28,9 +28,9 @@ Before starting the tutorial, verify the following:
 * You have `+node.js+` installed for front-end development and can install packages using `+npm install+` in your project.
 For information about installing node for your local operating system and package manager, see the link:https://nodejs.org/en/[Node] website.
 
-* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart{outfilesuffix}#download-and-install[Download and install].
+* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart-intro{outfilesuffix}#download-and-install[Download and install].
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/my-contacts.adoc:31]:
```suggestion
* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/local-quickstart{outfilesuffix}#download-and-install[Download and install].
```

File: modules/developers-guide/pages/tutorials/my-contacts.adoc [hunk 18]
```diff
@@ -28,9 +28,9 @@ Before starting the tutorial, verify the following:
...
 
-* You have installed the Visual Studio Code plugin for {proglang} as described in link:../../quickstart/quickstart{outfilesuffix}#install-vscode[Install the language editor plug-in] if you are using Visual Studio Code as your IDE.
+* You have installed the Visual Studio Code plugin for {proglang} as described in link:../../quickstart/quickstart-intro{outfilesuffix}#install-vscode[Install the language editor plug-in] if you are using Visual Studio Code as your IDE.
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/my-contacts.adoc:33]:
```suggestion
* You have installed the Visual Studio Code plugin for {proglang} as described in link:../../quickstart/local-quickstart{outfilesuffix}#install-vscode[Install the language editor plug-in] if you are using Visual Studio Code as your IDE.
```

File: modules/developers-guide/pages/tutorials/phonebook.adoc [hunk 19]
```diff
@@ -26,7 +26,7 @@ This program supports the following function calls:
 
 Before starting the tutorial, verify the following:
 
-* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart{outfilesuffix}#download-and-install[Download and install].
+* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart-intro{outfilesuffix}#download-and-install[Download and install].
```
Comment (lsgunnlsgunn) [inline modules/developers-guide/pages/tutorials/phonebook.adoc:29]:
```suggestion
* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/local-quickstart{outfilesuffix}#download-and-install[Download and install].
```

**Review by lsgunnlsgunn — CHANGES_REQUESTED** [review]
Some links need to go to a quickstart scenario rather than the intro

**Comment by lsgunnlsgunn** [comment]
Thanks for catching t˙ese broken links and typos. I've integrated your fixes in a different PR:
https://github.com/dfinity/docs/pull/292

**Comment by di-wu** [comment]
Alright, thanks @lsgunnlsgunn!

**Pull request closed without merging**